build.sbt

```diff
@@ -2,7 +2,9 @@ name := "PlayMapRouletteV2"
 
 version := "1.0"
 
-lazy val `playmaproulettev2` = (project in file(".")).enablePlugins(PlayScala)
+lazy val `playmaproulettev2` = (project in file("."))
+  .enablePlugins(PlayScala)
+  .enablePlugins(SbtWeb)
```

Comment:
@Crashfreak I am not sure if this is necessary - I had persistent problems with client side libraries (lessjs and jshint I believe) not being present, and including SbtWeb here resolved it. Any comments on this?